pallets/gear-eth-bridge/src/lib.rs

```diff
@@ -405,6 +551,9 @@ pub mod pallet {
                     // Setting zero queue root, keeping invariant of this key existence.
                     QueueMerkleRoot::<T>::put(H256::zero());
 
+                    // Transfer fees to treasury and refund to users.
+                    Self::transfer_fees();
```

Comment:
It may be too time consuming so I suggest to benchmark it and take the weight into consideration

Reply:
Added benchmark